### PR TITLE
Updating moving funds parameters

### PR DESCRIPTION
**Pull request by tomaszslabon**
Refs #https://github.com/keep-network/tbtc-v2/issues/152
This PR adds functionalities for updating parameters related to moving funds in Bridge.

### CODE REVIEW

File: solidity/contracts/bridge/Bridge.sol [hunk 1]
```diff
@@ -854,7 +859,28 @@ contract Bridge is Ownable, EcdsaWalletOwner {
...
+    ///        BTC transaction fee that is acceptable in a single moving funds
+    ///        transaction. This is a _total_ max fee for the entire moving
+    ///        funds transaction.
+    /// @param movingFundsTimeout New value of the moving funds timeout in
+    ///        seconds. It is the time after which the moving funds process can
+    ///        be reported as timed out. It is counted from the moment when the
+    ///        wallet was requested to move their funds and switched to the
+    ///        MovingFunds state.
+    /// @dev Requirements:
+    ///      - Moving funds timeout must be greater than zero
```
Comment (pdyraga) [inline solidity/contracts/bridge/Bridge.sol:874]: Shouldn't we require that movingFundsTxMaxTotalFee > 0 ?

File: solidity/test/bridge/Bridge.MovingFunds.test.ts [hunk 2]
```diff
@@ -49,8 +50,84 @@ describe("Bridge - Moving funds", () => {
...
+      treasury,
+      bank,
+      relay,
+      walletRegistry,
+      Bridge,
+      bridge,
+    } = await waffle.loadFixture(fixture))
+  })
+
+  describe("updateMovingFundsParameters", () => {
```
Comment (pdyraga) [inline solidity/test/bridge/Bridge.MovingFunds.test.ts:65]: `Bridge.Parameters.test.ts` please.

**Review by pdyraga — APPROVED** [review]

File: solidity/test/bridge/Bridge.MovingFunds.test.ts (hunk 2 above)
Comment (tomaszslabon) [reply solidity/test/bridge/Bridge.MovingFunds.test.ts:65]: Done in 49f3a5d7a6cabc18a383e4b72c7f4077cdf79499.

File: solidity/contracts/bridge/Bridge.sol (hunk 1 above)
Comment (tomaszslabon) [reply solidity/contracts/bridge/Bridge.sol:874]: Done in f52361d9df33814eeb9ed3ffa56562addb2c996b.

**Pull request merged**